src/ServerSide/InternalTypes.elm

```diff
@@ -108,23 +108,13 @@ decodeTagger =
         ]
 
 
-decodeKeyedNode : Json.Decode.Decoder NodeType
+decodeKeyedNode : Json.Decode.Decoder NodeRecord
 decodeKeyedNode =
-    Json.Decode.map NodeEntry
-        (Json.Decode.object4 NodeRecord
+    Json.Decode.object4 NodeRecord
         ("tag" := Json.Decode.string)
-        ("children" := Json.Decode.list decodeKeyedNodeType)
+        ("children" := Json.Decode.list (Json.Decode.at [ "_1" ] decodeNodeType))
```

Comment:
^^ this fixes an issue we ran into with decoders and recursion https://github.com/elm-lang/core/issues/664 it's related to https://github.com/elm-lang/elm-compiler/issues/873